CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+* Feat: Auto Transactions (#643)
```

Comment:
What about something more descriptive here like:

```suggestion
* Feat: Create transactions automatically for screen loading (#643)

```

Reply:
Ah, thought this should always mirror the PR name. Sounds good to me. 👍